### DISCUSSION TITLE
Progress bar on top of coverart dont work

### DISCUSSION THREAD

**Discussion by AndyFx12** [rern/rAudio#35] [Fixed Issues] [answered]
.. with todays update 202012281. 
Sliding works fine, though.

regards, Andy

**Comment by AndyFx12** [comment]
Issue still exists with 20201230
regards, Andy

**Comment by BV-Labs** [comment]
Same behavior on 20201230

**Comment by EarsofTin** [comment]
Using today's update - rAudio 1 • 20201230.  On small screen no progress bar until tap, progress bar then appears but does not advance.  Audio advances on tap.

**Comment by EarsofTin** [comment, marked as answer]
Working perfectly with 20201231 update.

Thanks Rern and Happy New Year to All!

**Reply by cYCL157** [reply]
working for me as well with 202021231 -- nice release to end the year!  thanks @rern !
